models/MethodDoc.php

```diff
@@ -41,5 +45,10 @@ public function __construct($reflector = null, $context = null, $config = [])
         $this->isStatic = $reflector->isStatic();
 
         $this->visibility = $reflector->getVisibility();
+
+        $lines = file($this->sourceFile);
+        for($i = $this->startLine - 1; $i <= $this->endLine - 1; $i++) {
+            $this->sourceCode .= substr($lines[$i], 4);
```

Comment:
`4` can be calculated dynamically. :thinking:

Reply:
What is "4" exactly?

Reply:
Number of spaces.

Reply:
Yes, it's the number of spaces we need to remove from the beginning of each line. I thought of existence of cases with different amount of spaces used for indentation, tabs instead of spaces and so on. But we discussed it with @samdark and decided to leave it as is now.